traffic_ops/traffic_ops_golang/crconfig/edgelocations.go

```diff
@@ -32,7 +32,7 @@ func makeLocations(cdn string, db *sql.DB) (map[string]tc.CRConfigLatitudeLongit
 
 	// TODO test whether it's faster to do a single query, joining lat/lon into servers
 	q := `
-select cg.name, t.name as type, cg.latitude, cg.longitude from cachegroup as cg
+select cg.name, cg.id, t.name as type, cg.latitude, cg.longitude from cachegroup as cg
```

Comment:
You need to select cg.fallback_to_closest here and scan the value into a `var fallbackToClosest *bool`. If it's null, set `BackupLocations.FallbackToClosest` below to false. Otherwise, set it to the dereferenced value.

Also, I noticed the CRConfig generated sets the value of `fallbackToClosest` to a string rather than a boolean. Since booleans are an actual type in Go, I'm thinking we should set the values to booleans rather than strings and make sure Traffic Router handles that properly.

Reply:
I took a look and found a bunch of bools which gets written as string in this particular part. For example=> sslEnabled.

Reply:
I think the string boolean thing might have stemmed from Perl not having real booleans. Now that it's been migrated to Go, we _could_ use real booleans in the JSON. But since there are a bunch of string bools in the CRConfig already, I guess it's better to be consistent and keep it as a string bool.